spring-kafka/src/main/java/org/springframework/kafka/listener/DeadLetterPublishingRecoverer.java

```diff
@@ -378,4 +506,84 @@ private String getStackTraceAsString(Throwable cause) {
 		return stringWriter.getBuffer().toString();
 	}
 
+	/**
+	 * Override this if you want different header names to be used
+	 * in the sent record.
+	 * @return the default Dlt header names
+	 */
+	protected HeaderNames getHeaderNames() {
+		return DeadLetterPublishingHeaderNamesBuilder
```

Comment:
We should make this a constant `DEFAULT_HEADERNAMES` to avoid building each time.

Reply:
Unresolved.